modules/behaviors/DataBehavior.js

```diff
@@ -860,10 +903,8 @@
      * @private
      */
     _dispose: function() {
+      Behavior.prototype._dispose.apply(this, arguments);
```

Comment:
There's no need to do the apply here if the parent is just a Torso.Behavior

Reply:
done